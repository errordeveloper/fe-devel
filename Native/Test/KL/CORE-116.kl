/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

struct Foo { Integer bar; } ;
struct Foo { Scalar bar; } ;

operator entry()
{
}
