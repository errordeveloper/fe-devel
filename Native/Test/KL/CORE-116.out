CORE-116.kl:6:1: error: type 'Foo' already registered with a different definition
